engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -740,6 +724,7 @@ class IrToTruffle(
         moduleScope.registerConversionMethod(toType, fromType, function)
       }
     })
+    moduleScope.build()
```

Comment:
Maybe this line got lost in the amount of changes but it is probably worth putting it in **bold**. This ensures that `ModuleScope.Builder` can't be modified.